### PR TITLE
Initial draft of statistical peer review post

### PR DESCRIPTION
**Pull request by noamross**

### CODE REVIEW

**Comment by yabellini** [comment]
This looks very nice and complete.  About the authors, I don't know what we used to do for the traditional review system. I am inclined to say that you add them with the role that corresponds to them.  It doesn't add a lot of content and it acknowledges all the people who in some way participate in this first important pilot.

**Comment by yabellini** [comment]
@noamross @mpadge should we proceed with this blog post?

**Comment by yabellini** [comment]
@steffilazerte can you so the technical check for this one so we can publish. Thanks!

**Review by mpadge — APPROVED** [review]

**Review by yabellini — APPROVED** [review]

**Review by steffilazerte — COMMENTED** [review]
Hi @noamross, Hi @mpadge,

How exciting! Thanks for writing this up. I have some technical suggestions (fixing the date, YAML coding and ALT text) as well as some editorial ones.

Some of the technically stuff does need to be fixed, but in general take what you like, and leave what you don't like!

I usually write an advertising tweet/toot, but sometimes Staff like to do it themselves. How do you feel? Would you like to do it or shall I?

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok -> **Will need to change, how about Tuesday Nov 29th?**
* [ ] alternative text of images is informative -> **See my suggestions for the badges**
* [ ] ~~Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")~~
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [ ] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor) -> **See my suggestions**

File: content/blog/2022-09-28-first-peer-reviewed-stats-packages/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+for editors and reviewers.To help them we've also built
+[automated infrastructure](https://ropensci.org/commcalls/dec2021-automation/) for checking submissions for compliance and producing metrics
+and diagnostics that help authors and reviewers navigate the review. 
+
+An enormous thank-you to the authors, editors, and reviewers who volunteered
+their time and helped piloted this new system! Also to our [statistical advisory
+board](https://stats-devguide.ropensci.org/) and all the community members that
+have provided input on our standards and processes.
+
+We're currently accepting packages in Bayesian statistics, expoloratory data analysis, machine-learning, regression and supervised learning, spatial and time-series statistics, and dimensionality reduction, clustering, and unsupervised learning. We will be expanding our standards to cover other areas such as network analysis in the the near future.
```
[the remark this answers is not in the document]
Comment (mpadge) [reply content/blog/2022-09-28-first-peer-reviewed-stats-packages/index.md:68]: Yes, you're right  @steffilazerte, that is indeed confusing! Our excuse is that all those "and"s represent each category as it is expressed - like "regression and supervised learning" - but the point here is to convey overall scope, and that is done much better with your edits. Thanks

[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2022-09-28-first-peer-reviewed-stats-packages/index.md]: That makes sense, perhaps semicolons next time (or this time?) to separate them out as categories?

**Comment by steffilazerte** [comment]
At the risk of pointing out things you already know, there are some comments that weren't resolved. 

You may not be done or you may not want to accept those comments (totally allowed!), but I'm pointing this out because GitHub hides a bunch of the middle comments, and lately they've been getting missed when I've been reviewing posts, so I just wanted to make sure!

**Comment by mpadge** [comment]
> At the risk of pointing out things you already know, there are some comments that weren't resolved.
> 
> You may not be done or you may not want to accept those comments (totally allowed!), but I'm pointing this out because GitHub hides a bunch of the middle comments, and lately they've been getting missed when I've been reviewing posts, so I just wanted to make sure!

Oh yeah, you're right! Sorry about that, and that is really annoying behaviour on GitHub's part. Especially because once they're unhidden and one is resolved, then they all automatically hide again straight away. Makes addressing them very laborious indeed!  Anyway, done now, thanks.

**Comment by steffilazerte** [comment]
I think if you look at the "Files Changed" panel, you can see the comments scattered throughout the scripts, which is a bit easier, but yes, definitely annoying! 

Thanks :grin:

Just need to update the date for tomorrow and I think we're good to go?

File: content/blog/2022-09-28-first-peer-reviewed-stats-packages/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: Our First Peer-Reviewed Statistical R Packages! 
+author: 
+  - Noam Ross
+  - Mark Padgham
+date: '2022-09-28T00:00:01-07:00'
```
Comment (mpadge) [inline content/blog/2022-09-28-first-peer-reviewed-stats-packages/index.md:6]:
```suggestion
date: '2022-11-30T00:00:01-07:00'
```

**Comment by steffilazerte** [comment]
Okay we're good for tomorrow!

Do either of you,  @noamross, @mpadge want to tweet/toot about the post? I'm happy to do so otherwise, but I know often staff like to do that themselves.

**Pull request merged**